docs/examples/frameworks/pytorch_setup/job.sh

```diff
@@ -16,9 +16,12 @@ module load anaconda/3
 
 # Creating the environment for the first time:
 # conda create -y -n pytorch python=3.9 pytorch torchvision torchaudio \
-#     pytorch-cuda=11.6 -c pytorch -c nvidia
+#     pytorch-cuda=11.7 -c pytorch -c nvidia
 
 # Activate the environment:
 conda activate pytorch
 
+# Fixes issues with MIG-ed GPUs.
+unset CUDA_VISIBLE_DEVICES
+
```

Comment:
This is not required with PyTorch 2.0. Can you please either add a note to specify this is useful for MIG + PyTorch < 2, or remove this workaround?

Reply:
Fixed in https://github.com/mila-iqia/mila-docs/pull/165/commits/ebbfced02061beb2bd6a458a2697b7b3a5d23a86